pkg/metrics/metrics.go

```diff
@@ -0,0 +1,26 @@
+package metrics
+
+import "github.com/prometheus/client_golang/prometheus"
+
+var (
+	registry                   = prometheus.NewRegistry()
+	storageBackendReconfigured = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace: "image_registry_operator",
+		Subsystem: "storage_backend",
```

Comment:
nit - could just be `storage` (aligns with the object in the CRD).

Reply:
Will fix it

Reply:
Done.